Report/sections/1_analysis/Learning.tex

```diff
@@ -21,4 +21,19 @@ \chapter{Video games and computer interest} \label{chap:analysis}
 Research from Georgia Institute of Technology has shown a positive correlation between playing video games and the interest in computer science. The research included a survey which 1116 students participated in; the results shows that 43\% of student claim that playing games have increased their interest in computing by a lot. 
 Even though the results from the study show a low interest in becoming a computer scientist (6.9\%), nearly 10\% (92 out of 1116 students) reported that playing video games sparked a “desire to make games”. 
 This means that a fair number of young students has the desire to make games without necessarily being interesting in learning or studying computer science. Therefore, a simple and easy programming language that enables the user to quickly create and explore the possibilities in game development could be in demand.
-As discussed in the previous section interest plays an important role in effective learning. Therefore learning a simple code language could be a good stepping stone to help understand a more complex languages.
\ No newline at end of file
+As discussed in the previous section interest plays an important role in effective learning. Therefore learning a simple code language could be a good stepping stone to help understand a more complex languages.
+
+\chapter{How to get more people starting with programming}\label{chap:analysis}
+
+This section will focus on how to get more people starting to program and what is the best ways to teach it. \newline
+
+Learn how to code is not easy, taking the first steps into this huge universe might seem like a daunting, if not intimidating task. All software is written in some kind of coding language and every coding language is unique, designed with a certain operating system, platform, coding style and intended use in mind. 
+That is why many have a hard time starting to program because the coding world can be complicated and confusing to get into. It takes years of pratice to learn everything necessary to get beyond just a basic level where you can write short programs that work. It takes hours of practice.
+There are plenty of free resources beginners can use to give themselves all the help they need. It can help them teach new techniques, and makes the learning process fun and exciting. 
```

Comment:
```suggestion
There are plenty of free resources beginners can use to give themselves all the help they need. It can help them teach new techniques, and make the learning process fun and exciting. 
```